app/models/reported_condition.rb

```diff
@@ -2,6 +2,8 @@
 
 # ReportedCondition
 class ReportedCondition < Condition
+  belongs_to :assessment, optional: true
```

Comment:
What is the condition that would result in a ReportedCondition that is not part of an assessment?

Reply:
It was causing these tests to fail at the time so I added 'optional' to it, but I'll go back and remove this and fix the tests, I don't think there would/should be a situation where a reported condition is not part of an assessment.

Reply:
fixed in: https://github.com/SaraAlert/SaraAlert/pull/157/commits/7970298fe8f0e63acc5d882863068009ba1bb082